test/symbol-test.js

```diff
@@ -96,8 +96,8 @@ tape("symbol.type(symbolOctagonAlt) generates a polygon with the specified size"
 
 tape("symbol.type(symbolOctagonAlt) generates the expected path", function(test) {
   var s = shape.symbol().type(extra.symbolOctagonAlt).size(function(d) { return d; });
-  test.pathEqual(s(0), "M0,0L0,0L0,0L0,0L0,0L0,0L0,0L0,0L0,0Z");
-  test.pathEqual(s(10), "M1.737172,0.719560L1.737172,0.719560L0.719560,1.737172L-0.719560,1.737172L-1.737172,0.719560L-1.737172,-0.719560L-0.719560,-1.737172L0.719560,-1.737172L1.737172,-0.719560Z");
+  test.pathEqual(s(0), "M0,0L0,0L0,0L0,0L0,0L0,0L0,0L0,0Z");
+  test.pathEqual(s(10), "M1.737172,0.719560L0.719560,1.737172L-0.719560,1.737172L-1.737172,0.719560L-1.737172,-0.719560L-0.719560,-1.737172L0.719560,-1.737172L1.737172,-0.719560Z");
```

Comment:
<img width="457" alt="1__vim" src="https://cloud.githubusercontent.com/assets/529247/25559125/ca7907ca-2d3d-11e7-8d45-29d7d40813f4.png">
And again.